src/libraries/System.Runtime.InteropServices/src/System/Runtime/InteropServices/Marshalling/ExceptionAsVoidMarshaller.cs

```diff
@@ -1,19 +1,14 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Numerics;
-using System.Text;
-using System.Threading.Tasks;
-
 namespace System.Runtime.InteropServices.Marshalling
 {
     /// <summary>
     /// Marshaller that swallows the exception.
     /// </summary>
+#pragma warning disable SYSLIB1057 // Marshaller type does not have the required shape
```

Comment:
This makes me sad. What parts are missing? I fear that people might accept this as "best practice". If we are sure this is appropriate, please add a `<remarks>` tag to indicate this isn't complete because... it is okay?

Reply:
The analyzer doesn't allow `void` as an unmanaged type, which this marshaller effectively has (and is correct in this scenario). We can extend the analyzer to allow this but I didn't want to block moving the product to public on that.